smac/smac_cli.py

```diff
@@ -83,12 +85,43 @@ def main_cli(self):
                     fn=traj_fn, cs=scen.cs)
                 initial_configs.append(trajectory[-1]["incumbent"])
 
+        # Restore state
```

Comment:
it should be also easier to write a unit test if we move this code to its own function